experiment-rollout.js

```diff
@@ -44,29 +44,25 @@ const data = {
         experimentType: 1,
         rolloutType: 1,
         priority: [
-            {
-                status: 1,
-                name: 'Verified bots (Server count wins name conflicts, other ones will get their discriminator appended)'
-            },
             {
                 status: 2,
                 name: 'Brands'
             },
             {
                 status: 2,
-                name: 'Discord staff'
+                name: 'Discord Staff'
             },
             {
-                status: 1,
-                name: 'Partnered and verified server owners (Includes old ones + some content creators)'
+                status: 2,
+                name: 'Partnered, subscription-enabled (100USD/mo+), and verified server owners (includes some old partners + some content creators)'
```

Comment:
> some old partners
I thought it's **all**?

Reply:
No, basically what happened is they took some kind of snapshot of all user IDs in the Partners server, that they then given access to Pomelos to. So that doesn't include *all* old partners.

Reply:
This also means some recently added partners might not have access to Pomelo.